lib/modules/uscore_v3.1.0_module.yml

```diff
@@ -52,8 +52,6 @@ test_sets:
         - USCore310ProvenanceSequence
         - USCoreR4DataAbsentReasonSequence
 sequence_requirements:
-  patient_ids:
```

Comment:
`sequence_requirements` here handles the new style sequence requirements. `patient_ids` is an old style sequence requirement and having it here caused issues (I think it was displayed twice, once with incorrect formatting).

Reply:
huh ok.  It looks like this one would work with the new style because its not unique like `confidential_client` which has a radio button iirc.  Either way its not the job of this PR to sort all that out.